Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/IndirectSampleChanger.py

```diff
@@ -0,0 +1,171 @@
+import os.path
+
+from mantid import config
+from mantid.api import *
+from mantid.kernel import *
+from mantid.simpleapi import *
+
+
+class IndirectSampleChanger(DataProcessorAlgorithm):
+    _plot = False
+    _save = False
+    _instrument = 'IRIS'
+    _number_runs = 1
+    _runs = None
+    _temperature = None
+    _q1_workspaces = None
+
+    def category(self):
+        return "Workflow\\MIDAS;PythonAlgorithms"
+
+    def summary(self):
+        return "Create elastic window scans for sample changer"
+
+    def PyInit(self):
+        self.declareProperty(name='Instrument', defaultValue='IRIS',
+                             validator=StringListValidator(['IRIS', 'OSIRIS']),
+                             doc='Instrument name')
+        self.declareProperty(name='Analyser', defaultValue='',
+                             validator=StringListValidator(['graphite', 'mica', 'fmica']),
+                             doc='Analyser bank used during run.')
+        self.declareProperty(name='Reflection', defaultValue='',
+                             validator=StringListValidator(['002', '004', '006']),
+                             doc='Reflection number for instrument setup during run.')
+
+        self.declareProperty(name='FirstRun', defaultValue=1,
```

Comment:
The `defaultValue` here seems a little confusing. Are they set to these just so that they are integer properties?

If so can we do something like `self.declareProperty("FirstRun",-1,InttBoundedValidator(lower=0)) (
https://www.mantidproject.org/Algorithm_Property_Validation) instead?